netutils/config/parser.py

```diff
@@ -695,12 +695,36 @@ def __init__(self, config):
         self.uncommon_data = self._get_uncommon_lines(config)
         super(FortinetConfigParser, self).__init__(config)
 
+    def is_end_next(self, line):  # pylint: disable=no-self-use
+        """Determine if line has 'end' or 'next' in it.
+
+        Args:
+            line (str): A config line from the device.
+
+        Returns:
+            bool: True if line has 'end' or 'next', else False.
+
+        Example:
+            >>> FortinetConfigParser("config system virtual-switch").is_comment("config system virtual-switch")
+            False
+            >>> FortinetConfigParser("end").is_comment("end")
+            True
+            >>>
+        """
+        for end_next in ["end", "next"]:
+            if line.lstrip() == end_next:
+                return True
+        return False
+
     def _parse_out_offending(self, config):  # pylint: disable=no-self-use
         """Preprocess out strings that offend the normal spaced configuration syntax.
 
         Args:
             config (str): full config as a string.
         """
+        # This will grab everything between quotes after the 'set buffer' sub-command.
+        # Its explicitly looking for "\n to end the captured data.  This is to support html
+        # data that is supported in Fortinet config with double quotes within the html.
```

Comment:
Still need to document in someway that get to read the docs that we will not an html syntax that somehow starts the line with a double quote.